python/taichi/ui/scene.py

```diff
@@ -134,8 +131,5 @@ def particles(self,
         vbo_info = get_field_info(vbo)
         super().particles(vbo_info, has_per_vertex_color, color, radius)
 
-    def point_light(self, pos, color):
+    def point_light(self, pos, color):  # pylint: disable=W0235
```

Comment:
Why handling this differently :-) ?

Reply:
https://github.com/taichi-dev/taichi/runs/4191773093

@k-ye CI fails, but I'm not familiar with pybind11.